src/Symfony/Component/String/Data/WcswidthDataGenerator.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\String\Data;
+
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Component\String\Exception\RuntimeException;
+use Symfony\Component\VarExporter\VarExporter;
+
+/**
+ * @internal
+ */
+final class WcswidthDataGenerator
```

Comment:
Should be named `WcwidthDataGenerator` - actually, all "s" should be removed from the PR I think, to use only "wcwidth"

Reply:
Also I'd suggest moving this in the Resources folder, and adding it to the gitattributes as export-ignore